pkg/process/config/config.go

```diff
@@ -95,19 +95,18 @@ type WindowsConfig struct {
 //
 // Deprecated. Use `pkg/config` directly.
 type AgentConfig struct {
-	Enabled                   bool
-	HostName                  string
-	APIEndpoints              []apicfg.Endpoint
-	QueueSize                 int // The number of items allowed in each delivery queue.
-	RTQueueSize               int // the number of items allowed in real-time delivery queue
-	ProcessQueueBytes         int // The total number of bytes that can be enqueued for delivery to the process intake endpoint
-	Blacklist                 []*regexp.Regexp
-	Scrubber                  *DataScrubber
-	MaxPerMessage             int
-	MaxCtrProcessesPerMessage int // The maximum number of processes that belong to a container for a given message
```

Comment:
Noice 👍🏻

Reply:
It's not bloat, it's code, someone's hard work 😄